public/js/src/module/connection.js

```diff
@@ -541,12 +547,7 @@ function _request(url, method = 'POST', data = {}) {
 
     return fetch(url, options)
         .then(_throwResponseError)
-        .then((response) => response.json())
-        .catch((data) => {
-            const error = new Error(data.message);
-            error.status = data.status;
-            throw error;
-        });
```

Comment:
The error handler didn't seem to do anything (helpful), and it seemed within scope to clean that up.